huggle/wikiedit.cpp

```diff
@@ -106,6 +106,24 @@ bool WikiEdit::FinalizePostProcessing()
         return false;
     }
 
+    if (this->qCategories != nullptr && this->qCategories->IsProcessed())
+    {
+        if (this->qCategories->IsFailed())
+        {
+            Syslog::HuggleLogs->ErrorLog("Unable to fetch categories for page " + this->Page->PageName + ": " + this->qCategories->GetFailureReason());
```

Comment:
I think this was just following the existing pattern for other post-process queries